diff --git a/node_modules/@capacitor-community/sqlite/Package.swift b/node_modules/@capacitor-community/sqlite/Package.swift
index 97b9e7d..d77ddaa 100644
--- a/node_modules/@capacitor-community/sqlite/Package.swift
+++ b/node_modules/@capacitor-community/sqlite/Package.swift
@@ -10,7 +10,7 @@ let package = Package(
             targets: ["CapacitorSQLitePlugin"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", branch: "8.0.0"),
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "8.0.0"),
         .package(url: "https://github.com/sqlcipher/SQLCipher.swift.git", from: "4.14.0"),
         .package(url: "https://github.com/weichsel/ZIPFoundation.git", from: "0.9.0")
     ],
